generators/app/templates/static/scss/tools/_tools.interaction.scss

```diff
@@ -7,18 +7,20 @@
  * @include hover{}
  */
 @mixin hover() {
-    @media (hover: none) {
-        -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
-        &:active {
-            /* 1 */
-            @content;
+    @include mq(null, lg) {
+        @media (hover: none) {
+            -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+            &:active {
+                /* 1 */
+                @content;
+            }
         }
```

Comment:
ja bi maknuo ovo skroz, zivcira me taj active, triggeraju nam se "hover" statovi na mobitelu zbog tog active

Reply:
zato sam ja wrappao to u (null, lg), bas iz tog razloga, mislim da sam to sad na meetu bas specificno i objasnjavao

Reply:
jesam, makar ja sam cak za raspravu da skroz maknemo taj :active, i onda nemo problema sa touch uređajima